layers/poi/class.sql

```diff
@@ -46,24 +48,28 @@ RETURNS TEXT AS $$
         WHEN subclass IN ('books','library') THEN 'library'
         WHEN subclass IN ('university','college') THEN 'college'
         WHEN subclass IN ('hotel','motel','bed_and_breakfast','guest_house','hostel','chalet','alpine_hut','camp_site') THEN 'lodging'
-        WHEN subclass IN ('chocolate','confectionery') THEN 'ice_cream'
+        WHEN subclass IN ('chocolate','confectionery', 'ice_cream') THEN 'ice_cream'
         WHEN subclass IN ('post_box','post_office') THEN 'post'
-        WHEN subclass IN ('cafe') THEN 'cafe'
+        WHEN subclass IN ('cafe', 'coffee') THEN 'cafe'
         WHEN subclass IN ('school','kindergarten') THEN 'school'
-        WHEN subclass IN ('alcohol','beverages','wine') THEN 'alcohol_shop'
+        WHEN subclass IN ('alcohol','beverages','wine', 'nightclub') THEN 'alcohol_shop'
         WHEN subclass IN ('bar','nightclub') THEN 'bar'
-        WHEN subclass IN ('marina','dock') THEN 'harbor'
-        WHEN subclass IN ('car','car_repair','taxi') THEN 'car'
-        WHEN subclass IN ('hospital','nursing_home', 'doctors', 'clinic') THEN 'hospital'
+        WHEN subclass IN ('marina','dock', 'boat_rental', 'boat_sharing', 'ferry_terminal') THEN 'harbor'
+        WHEN subclass IN ('car','car_repair','taxi', 'fuel', 'parking', 'car_wash', 'car_rental', 'charging_station', 'parking_space') THEN 'car'
+        WHEN subclass IN ('hospital','nursing_home', 'doctors', 'clinic', 'dentist', 'pharmacy', 'veterinary') THEN 'hospital'
```

Comment:
hospital is very highly ranked. If we group dentist and veterinary with them, they would be over ranked.

Even the doctors offices has been removed of this class for this exact reason : https://github.com/openmaptiles/openmaptiles/commit/d48b623eda6e429f3083b9ade9a1841ed4d759d2#diff-fe7aa15546085ac36d427f99e23a5fbd